clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -137,6 +136,17 @@ abstract class OpenstackClientProvider {
 
 
   /**
+   * Deletes a security group.
+   *
+   * @param securityGroupId id of the security group
+   */
+  void deleteSecurityGroup(String securityGroupId) {
+    handleRequest(AtomicOperations.DELETE_SECURITY_GROUP) {
```

Comment:
No ... just redesigning it a bit to be operation agnostic so we can use between caching and operations